### ISSUE TITLE
Sidebar stays collapsed until all js has been loaded in.

### ISSUE THREAD

**Issue by Daniel-I-Am** [HZ-HBO-ICT/Laravel-CoreUI#27]
# Issue description
While the sidebar is collapsed (how it is by default) everything looks fine. If you expand the sidebar and then refresh the page, it will switch between collapsed and expanded on a reload. By limiting your connection through your browser's network throttle (this can be found inside the developer tools on Chrome and FF), you can see the effect a little better. The sidebar stays collapsed until all files on the page have been loaded in, only then does it take your preference and expand.

# Steps to reproduce
```sh
laravel new coreui-issue-test
cd coreui-issue-test/
composer require hz-hbo-ict/laravel-core-ui
php artisan vendor:publish --provider="HzHboIct\LaravelCoreUI\ServiceProvider" --tag=assets
```
- Replace the default `welcome.blade.php` with:
```php
@extends('coreui::master')
```
- View the home page
- Expand the sidebar
- Reload the page

Most likely caused by 036d5159bf57dff8a3c74d10656c18114c3ed873

**Comment by Daniel-I-Am** [comment]
Easiest fix for this would probably be to add something like
```php
@if(request()->cookie('remember_sidebar'))
<body class="app header-fixed sidebar-fixed sidebar-lg-show">
@else
<body class="app header-fixed sidebar-fixed sidebar-lg-show sidebar-minimized brand-minimized">
@endif
```
to [the master blade body opening tag](https://github.com/HZ-HBO-ICT/Laravel-CoreUI/blob/6cc03f1536cf1eb6151f13525a9dcb59d82d8154/resources/views/master.blade.php#L17).
That does require laravel to *not* encrypt the `remember_sidebar` cookie, otherwise this will fail.
The default EncryptCookies middleware would have to be changed in order to change the whitelist.
```php
    protected $except = [
        'remember_sidebar'
    ];
```

**Comment by Lucky-Loek** [comment]
As commented on PR #28, I'm working on a complete client-side solution for this. It shouldn't be the responsibility of the server-side code to show the correct menu bar. Cookies probably weren't the best choice in the first place. Leaving this issue and PR open as mentioned in PR.

**Comment by Lucky-Loek** [comment]
Fixed in v2.7.1
